### PR TITLE
[FSSDK-9358] chore: add flutter specific client name support to track event

### PR DESCRIPTION
**Pull request by muzahidul-opti**
## Summary
- Client name and version added to track the event sent by flutter sdk.

## Test plan
- All previous tests should be passed.

## Issues
- [FSSDK-9358](https://jira.sso.episerver.net/browse/FSSDK-9358)

### CODE REVIEW

File: lib/package_info.dart [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Purpose: Package information for the Optimizely Flutter SDK
+// Note: This info should be synced with pubspec.yaml
+
+class PackageInfo {
+  static const String name = 'optimizely_flutter_sdk';
+  static const String version = '2.0.0-beta';
```
Comment (jaeopt) [inline lib/package_info.dart:6]: why beta here?

File: lib/src/optimizely_client_wrapper.dart [hunk 2]
```diff
@@ -53,6 +54,7 @@ class OptimizelyClientWrapper {
   static Map<String, Map<int, LogEventNotificationCallback>>
       logEventCallbacksById = {};
   static Map<String, Map<int, MultiUseCallback>> configUpdateCallbacksById = {};
+  static String sdkVersion =  "";
```
Comment (jaeopt) [inline lib/src/optimizely_client_wrapper.dart:57]: what is this for in addition to one in line 71?

File: lib/src/utils/utils.dart [hunk 3]
```diff
@@ -15,11 +15,12 @@
 ///**************************************************************************/
 
 import 'dart:io' show Platform;
-
 import 'package:optimizely_flutter_sdk/src/user_context/optimizely_user_context.dart';
 import 'package:optimizely_flutter_sdk/src/utils/constants.dart';
 import 'package:optimizely_flutter_sdk/src/data_objects/log_level.dart';
 
+
+
```
Comment (jaeopt) [inline lib/src/utils/utils.dart:23]: nit - we may need lint :)

**Review by jaeopt — APPROVED** [review]
Looks good! Can we double check -
1. logx handles those new client names with "/" ok.
2. iOS privacy file is handled ok in flutter sdk too when building for iOS.

File: lib/package_info.dart (hunk 1 above)
Comment (muzahidul-opti) [reply lib/package_info.dart:6]: Will update in the release branch

File: lib/src/optimizely_client_wrapper.dart (hunk 2 above)
Comment (muzahidul-opti) [reply lib/src/optimizely_client_wrapper.dart:57]: That's is my bad, will remove it.

**Comment by muzahidul-opti** [comment]
> Member



> Looks good! Can we double check -
> 
> 1. logx handles those new client names with "/" ok.
> 2. iOS privacy file is handled ok in flutter sdk too when building for iOS.

1. Got some data in the report page but can't be able to verify it.
2. This issue is still open for flutter https://github.com/flutter/flutter/issues/131940. May be we need to wait for the solution.

File: lib/src/utils/utils.dart (hunk 3 above)
Comment (muzahidul-opti) [reply lib/src/utils/utils.dart]: fixed

**Pull request merged**